LoRaEngine/deployment.ci.lns.eflow.template.json

```diff
@@ -114,6 +114,31 @@
             "status": "running",
             "restartPolicy": "always",
             "version": "1.0"
+          },
+          "metricscollector": {
+            "settings": {
+              "image": "mcr.microsoft.com/azureiotedge-metrics-collector:1.0"
+            },
+            "type": "docker",
+            "env": {
+              "ResourceId": {
+                "value": "$IOT_HUB_RESOURCE_ID"
+              },
+              "UploadTarget": {
+                "value": "AzureMonitor"
+              },
+              "LogAnalyticsWorkspaceId": {
+                "value": "$LOG_ANALYTICS_WORKSPACE_ID"
+              },
+              "LogAnalyticsSharedKey": {
+                "value": "$LOG_ANALYTICS_SHARED_KEY"
+              },
+              "MetricsEndpointsCSV": {
+                "value": "http://edgeHub:9600/metrics,http://edgeAgent:9600/metrics,http://LoRaWanNetworkSrvModule:5000/metrics"
```

Comment:
For the `azuredeploy.json` we chose to not collect prometheus metrics via the collector module - if we deployed a Log Analytics workspace we used to export the metrics directly to azure monitor. this will make sure that the metrics are also accessible with the metrics explorer. Do we still want to collect the metrics on the LNS as well?

Reply:
As discussed orally, maybe having the CI monitor the two is a good idea.